packages/amplify-cli-core/src/feature-flags/featureFlags.ts

```diff
@@ -502,6 +502,12 @@ export class FeatureFlags {
         defaultValueForExistingProjects: false,
         defaultValueForNewProjects: true,
       },
+      {
+        name: 'defaultSecondaryIndex',
```

Comment:
maybe name this `secondaryKeyAsGSI`? To me `defaultSecondaryIndex` doesn't really convey the right meaning